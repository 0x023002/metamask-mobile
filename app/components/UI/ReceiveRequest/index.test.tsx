import React from 'react';
import { shallow } from 'enzyme';
import ReceiveRequest from './';
import configureMockStore from 'redux-mock-store';
import { Provider } from 'react-redux';
<<<<<<< Updated upstream
=======
import initialBackgroundState from '../../../util/test/initial-background-state.json';
>>>>>>> Stashed changes

const mockStore = configureMockStore();
const initialState = {
  engine: {
<<<<<<< Updated upstream
    backgroundState: {
      PreferencesController: { selectedAddress: '0x' },
      NetworkController: { network: '1', provider: { ticker: 'ETH' } },
    },
=======
    backgroundState: initialBackgroundState,
>>>>>>> Stashed changes
  },
  modals: {
    receiveAsset: {},
  },
  user: {
    seedphraseBackedUp: true,
  },
<<<<<<< Updated upstream
	engine: {
		backgroundState: {
			PreferencesController: { selectedAddress: '0x' },
			NetworkController: { network: '1', provider: { ticker: 'ETH' } },
		},
	},
	modals: {
		receiveAsset: {},
	},
	user: {
		seedphraseBackedUp: true,
	},
=======
  fiatOrders: {
    networks: [
      {
        active: true,
        chainId: 1,
        chainName: 'Ethereum Mainnet',
        nativeTokenSupported: true,
      },
    ],
  },
>>>>>>> Stashed changes
};
const store = mockStore(initialState);

describe('ReceiveRequest', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <ReceiveRequest />
      </Provider>,
    );
<<<<<<< Updated upstream
    expect(wrapper.dive()).toMatchSnapshot();
=======
    expect(wrapper).toMatchSnapshot();
>>>>>>> Stashed changes
  });
});
