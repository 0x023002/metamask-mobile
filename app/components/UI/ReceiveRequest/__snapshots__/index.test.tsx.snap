// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReceiveRequest should render correctly 1`] = `
<<<<<<< Updated upstream
<ReceiveRequest
  network="1"
  protectWalletModalVisible={[Function]}
  receiveAsset={Object {}}
  seedphraseBackedUp={true}
  selectedAddress="0x"
  showAlert={[Function]}
  ticker="ETH"
  toggleReceiveModal={[Function]}
/>
=======
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <Component />
</ContextProvider>
>>>>>>> Stashed changes
`;
